### PR TITLE
feat: add ilqp

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by michielbdejong** [comment]
console.log statements are used directly for logging. we can switch this to debug, I guess, but haven't gotten round to that yet, as it's really just a short script.

**Comment by dappelt** [comment]
installation fails with:

```sh
/ilp-plugin-xrp-escrow#3fadeb4 Command failed: /usr/local/bin/git checkout 3fadeb4
471 silly fetchPackageMetaData error: pathspec '3fadeb4' did not match any file(s) known to git.
```

File: index.js [hunk 1]
```diff
@@ -8,6 +8,47 @@ const plugin = new Plugin({
   btpUri: 'btp+wss://interfaucet:' + process.env.TOKEN + '@amundsen.michielbdejong.com/api/17q3',
 })
 
+function getQuote(ippBuf) {
```
Comment (dappelt) [inline index.js:11]: Why don't you use the `ilp` module for quoting?

Comment (michielbdejong) [reply index.js:11]: I could copy this function into the `ilp` module and then include that dependency, but this is such a small function that I think it's better to keep it the way it is.

**Review by dappelt — APPROVED** [review]

**Pull request merged**